lib/maplibre-compose/src/androidMain/kotlin/dev/sargunv/maplibrecompose/core/layer/RasterLayer.kt

```diff
@@ -12,35 +14,35 @@ internal actual class RasterLayer actual constructor(id: String, actual val sour
   Layer() {
   override val impl = MLNRasterLayer(id, source.id)
 
-  actual fun setRasterOpacity(opacity: Expression<Number>) {
+  actual fun setRasterOpacity(opacity: FloatExpression) {
     impl.setProperties(PropertyFactory.rasterOpacity(opacity.toMLNExpression()))
   }
 
-  actual fun setRasterHueRotate(hueRotate: Expression<Number>) {
+  actual fun setRasterHueRotate(hueRotate: IntExpression) {
```

Comment:
```suggestion
  actual fun setRasterHueRotate(hueRotate: FloatExpression) {
```

see https://github.com/maplibre/maplibre-native/blob/72d2c70130fcd4e320cf623f9bec811108b6e201/src/mbgl/style/layers/raster_layer_properties.hpp#L35

(even though it may uncommon, the hue is essentially an angle)